xarray/tests/test_backends.py

```diff
@@ -2469,6 +2371,33 @@ def test_ENVI_tags(self):
                 assert isinstance(rioda.attrs['map_info'], basestring)
                 assert isinstance(rioda.attrs['samples'], basestring)
 
+    def test_no_mftime(self):
+        # rasterio can accept "filename" urguments that are actually urls,
+        # including paths to remote files.
+        # In issue #1816, we found that these caused dask to break, because
+        # the modification time was used to determine the dask token. This
+        # tests ensure we can still chunk such files when reading with
+        # rasterio.
+        with create_tmp_geotiff(8, 10, 3, transform_args=[1, 2, 0.5, 2.],
+                                crs='+proj=latlong') as (tmp_file, expected):
+            with mock.patch('os.path.getmtime', side_effect=OSError):
+                with xr.open_rasterio(tmp_file, chunks=(1, 2, 2)) as actual:
+                    import dask.array as da
+                    self.assertIsInstance(actual.data, da.Array)
+                    assert_allclose(actual, expected)
+
+    @network
+    def test_http_url(self):
+        # more examples urls here
+        # http://download.osgeo.org/geotiff/samples/
+        url = 'http://download.osgeo.org/geotiff/samples/made_up/ntf_nord.tif'
```

Comment:
I forgot about the `@network` decorator, this can be quite useful for other tests as well...